chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,963 @@
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"const expresult = pm.response.json()\r",
+									"pm.test(\"/{certifiedProductId} end point response should have all RWT fields\", function () {\r",
+									"    pm.expect(expresult).to.have.property('rwtPlansUrl');\r",
+									"    pm.expect(expresult).to.have.property('rwtPlansCheckDate');\r",
+									"    pm.expect(expresult).to.have.property('rwtResultsUrl');\r",
+									"    pm.expect(expresult).to.have.property('rwtResultsCheckDate');\r",
+									"    pm.expect(expresult).to.have.property('rwtEligibilityYear');\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-Key",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/certified_products/10295",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"certified_products",
+								"10295"
+							]
+						},
+						"description": "GET /{certifiedProductId} end point should return status 200 and response should have all RWT fields for valid API Key"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /{certifiedProductId}/measures- valid API Key",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"const certifiedProductId= pm.iterationData.get(\"certifiedProductId\");\r",
+									"pm.environment.set(\"certifiedProductId\",\"certifiedProductId\");"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"/{certifiedProductId}/measures end point should return status 200 for valid API Key\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-KEY",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/certified_products/9804/measures",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"certified_products",
+								"9804",
+								"measures"
+							]
+						},
+						"description": "GET /{certifiedProductId}/measures end point returns response status 200 for valid API Key and valid certifiedProductId"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /{certifiedProductId}/measures - Invalid API Key",
```

Comment:
I am not sure how much value it is adding testing each end point with invalid and no api key. If we want to test these, i would suggest to make it data driven and also put the end point in the folder under main controller to group all the tests. Adding folder would trigger change in newman file too.

Same comments for all the requests below "GET : Certified_products: non-data driven tests" folder. Need to rearrange tests precisely under certified products controller.

Reply:
Rearranged tests under "GET : Certified_products: non-data driven tests" folder